v-next/hardhat-ethers/src/internal/hardhat-ethers-provider/hardhat-ethers-provider.ts

```diff
@@ -133,7 +133,7 @@ export class HardhatEthersProvider implements HardhatEthersProviderI {
       const accounts: string[] = await accountsPromise;
       if (address >= accounts.length) {
         throw new HardhatError(
-          HardhatError.ERRORS.ETHERS.ACCOUNT_INDEX_OUT_OF_RANGE,
+          HardhatError.ERRORS.HARDHAT_ETHERS.GENERAL.ACCOUNT_INDEX_OUT_OF_RANGE,
```

Comment:
This makes sense to me, but can I check why the change from `ETHERS` to `HARDHAT_ETHERS` is necessary?

Reply:
It was just how Pato wrote up the example in our design call so I copied it over